astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -41,13 +44,30 @@ class galactocentric_frame_defaults(ScienceState):
     ``Galactocentric`` or ``Galactocentric()`` with no explicit arguments.
 
     This class controls the parameter settings by specifying a string name,
-    which can be one of:
+    with the following pre-specified options:
 
     - 'pre-v4.0': The current default value, which sets the default frame
       attribute values to their original (pre-astropy-v4.0) values.
     - 'v4.0': The attribute values as updated in Astropy version 4.0.
     - 'latest': An alias of the most recent parameter set (currently: 'v4.0')
 
+    Alternatively, user-defined parameter settings may be registered, with
+    :meth:`~astropy.coordinates.galactocentric_frame_defaults.register`,
+    and used identically as pre-specified parameter sets. At minimum,
+    registrations must have unique names and a dictionary of parameters
+    with keys "galcen_coord", "galcen_distance", "galcen_v_sun", "z_sun",
+    "roll". See :ref:`this example <galactocentric-frame-defaults-registry-ex>`.
```

Comment:
I think a cross-link in a docstring is not very helpful - these are mostly going to be seen inside `ipython` or so.  I'd just delete this, but if anything, just put "See examples below".

Reply:
Very true. implementing.